src/Shared Components/Pagination.js

```diff
@@ -0,0 +1,32 @@
+const Pagination = ({ page, setPage, list, limit }) => {
+    const getMaxPageNumber = () => {
+      const result = Math.ceil(list.length / limit);
+      if (result < 1) return 1;
+      return result;
+    };
+  
+    const handlePageChange = (e, type) => {
+      const maxNumber = getMaxPageNumber(limit);
+      if (type === "next" && page < maxNumber) setPage(page + 1);
+      if (type === "previous" && page > 1) setPage(page - 1);
+      if (type === "input") setPage(e.target.value);
+    };
+  
+    return (
+      <div>
+        <button onClick={(e) => handlePageChange(e, "previous")}>&#x2190;</button>
```

Comment:
What about to not show prev arrow when current page is equal 1 and next arrow when current page is equal last?

```suggestion
{ page > 1 && <button onClick={(e) => handlePageChange(e, "previous")}>&#x2190;</button> }
```